Packs/Sysdig/Integrations/Sysdig-Response-Actions/SysdigResponseActions.py

```diff
@@ -0,0 +1,313 @@
+"""
+Sysdig Response Actions Integration
+"""
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+
+import re
+from typing import Any, Dict, Tuple
+import urllib3
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+""" CONSTANTS """
+
+SYSTEM_CAPTURES_REQUIRED_FIELDS = ["container_id", "host_name", "capture_name", "agent_id", "customer_id", "machine_id"]
+RESPONSE_ACTIONS_REQUIRED_FIELDS = ["actionType", "callerId"]
+RESPONSE_ACTIONS_PARAMS = {
+    "FILE_QUARANTINE": ["path.absolute", "container.id"],
+    "KILL_PROCESS": ["process.id", "host.id"]
+}
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"  # ISO8601 format with UTC, default in XSOAR
+
+""" CLIENT CLASS """
+class Client(BaseClient):
+    """Client class to interact with the service API
+
+    This Client implements API calls, and does not contain any XSOAR logic.
+    Should only do requests and return data.
+    It inherits from BaseClient defined in CommonServerPython.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    For this  implementation, no special attributes defined
+    """
+
+    def __init__(
+        self,
+        base_url: str,
+        verify: bool,
+        headers: Dict[str, str],
+        proxy: bool,
+        ok_codes: Tuple[int] = (200, 201, 202)
+    ):
+        self.base_url = base_url
+        self.verify = verify
+        self.headers = headers
+        self.proxy = proxy
+        self.ok_codes = ok_codes
+        super().__init__(base_url=base_url, verify=verify, headers=headers, proxy=proxy, ok_codes=ok_codes)
+
+
+    def call_sysdig_api(self, method: str = "GET", url_suffix: str = "/secure/response-actions/v1alpha1/action-executions", params: dict = None, data: dict = None, json_data: dict = None, resp_type: str = 'json', full_url: str = None) -> Dict[str, Any]:
```

Comment:
```suggestion
    def call_sysdig_api(self, method: str = "GET", url_suffix: str = "/secure/response-actions/v1alpha1/action-executions", params: dict = None, data: dict = None, json_data: dict = None, resp_type: str = 'json', full_url: str = None) -> Dict[str, Any] | bytes | str:
```
As it may return str or bytes as well when you set the res_type to content